_posts/2017-04-04-things-the-rust-community-team-does.md

```diff
@@ -0,0 +1,101 @@
+
+The same goes for conferences, where we have a lot of experience in running fair, cheap and accessible conferences, knowledge which we happily share.
+
+We also help out other communities at that. This comes directly from our [initial goals]: "Coordination with other technology communities, regardless of language. We should be encouraging cross-polination between Rust and other languages, in both directions."
+
+## Encouraging
+
+This might be the most subtle thing we do, as it's rarely much more then a couple [of words](https://www.reddit.com/r/rust/comments/61avqf/want_to_join_the_rust_docs_team/dfd45md/) or a link. Projects like Rust often suffer from the problem that people don't feel qualified to be involved in certain things. Often, that happens because they feel to "rookie", but also because they feel a certain team is a closed group. This can be easily fixed by setting these misconceptions right. Also, we encouraged a group of people to run a [RustBridge](https://github.com/rust-community/rustbridge).
+
+## Switchboard
+
+Another surprisingly huge part of our work is what Manish called the "switchboard". We make sure the right people get in touch. This is often as easy as knowing two people in the same city and introducing them. While the action is easy, it needs a good knowledge of the community.
+
+It allows us to recommend our [community-team@rust-lang.org](mailto:community-team@rust-lang.org) email address as a starting point for all your questions about Rust. The service here is that instead of potential newcomers figuring out who they should speak to, we do the research for them.
+
+
+## Inventory and Documentation
+
+We do keep an inventory of things we use for promotion. This is currently rather small (table blankets with a Rust logo), but in general, it is our job. An outstanding issue is figuring out who owns [all the accounts](https://github.com/rust-community/team/issues/88).
+
+## Run Conferences
+
+Two conferences (RustFest and Rust Belt Rust) are run by community team members. They also serve as an example to see what can be done with conferences. RustFest and Rust Belt Rust both share a workshop component, but in different fashions. Both are very affortable. RustFest also wants to innovate on [accessibility standards](http://2016.rustfest.eu/accessibility/) and can be approached if you search consultation.
+
+We'd like to set community standards here.
+
+## Evangelism
+
+This is mostly the usual "fly around and talk at conferences" fare. Sometimes, we have stands on most conferences. The interesting part of FOSS evangelism, though, is that we don't want to sell you a support contract later. We're genuinely interested in you!
+
+We recommend people to try Rust, mostly by showing curious people what we have by showing them the way to the right material. We love assessing solutions, so sometimes, we also inform about other options.
+
+## Criticism
```

Comment:
I think you mean "context", not "contest".

I think this could be merged with "Identify Upcoming Issues".